lib/skate_web/templates/unauthorized/restricted_environment.html.eex

```diff
@@ -0,0 +1,6 @@
+
+<div>
+    Hi there!<br>
+    The URL you followed is actually for Skate's test environment, meant for testing purposes only. Please do not use this URL; use <a href="https://skate.mbta.com">skate.mbta.com</a> instead.<br> If you're unsure how this happened or are confused, please reach out to the Skate team at
+    <a href = "mailto: skateteam@mbta.com">skateteam@mbta.com</a>.
```

Comment:
**nitpick:** Is it correct to have a space after the `:` and before the email address?

Reply:
Seems to work either way! I pulled it from a [w3 example](https://www.tutorialspoint.com/html/html_email_links.htm) but can change if it is more conventional to omit the space!

Reply:
If it works I'd say it's fine either way.